src/test/java/com/at/stepdefinitions/AdminSteps.java

```diff
@@ -0,0 +1,26 @@
+package com.at.stepdefinitions;
+
+import com.at.factories.DriverFactory;
+import com.at.pageobject.AdminPage;
+import com.at.pageobject.LogInPage;
+
+import io.cucumber.java.en.And;
+
+public class AdminSteps extends DriverFactory {
+	
+	@And("click on Users button")
+	public void click_on_users_button() throws Throwable {
+		
+		new AdminPage(driver).clickOnUserButton();
+		
+	}
+	@And("show all users")
+	public void show_all_users() throws Throwable {
+
+		Thread.sleep(5000);
+		new AdminPage(driver).setShowNumber();
```

Comment:
Here you should be passing values to this method. For example setShowNumber(numberOfUsers)